content/vsphere-human-readable-names.md

```diff
@@ -22,14 +21,13 @@ With this feature enabled, when a new vm is created, it will be assigned to a na
       vm_folder: ((vcenter_vms))
     password: ((vcenter_password))
     user: ((vcenter_user))
-    enable_human_readable_name: false
+    enable_human_readable_name: true
 ```
 
 2. Deploy the Director
 
 
-# More about human readable names
-
+## More about human readable names
 1. Currently vSphere CPI only support names in ASCII characters.
 
 2. The total max length kept for instance group name and deployment name is 65. If original names are longer, names will be trimmed. A 25 digits space is  guaranteed for deployment name part if need to trim; for instance group name, the number is 40.
```

Comment:
You can remove line `A 25 digits space...`  => This level of implementation detail is not necessary here.

Reply:
Done.